checkov/terraform/checks/resource/kubernetes/PrivilegedContainer.py

```diff
@@ -0,0 +1,34 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class PrivilegedContainersPSP(BaseResourceCheck):
+
+    def __init__(self):
+        # CIS-1.3 1.7.1
+        # CIS-1.5 5.2.1
+        name = "Do not admit privileged containers"
+        id = "CKV_K8S_2"
```

Comment:
now I double checked and this check ID is just for PSP, the one for container is `CKV_K8S_16`